package.json

```diff
@@ -25,6 +25,7 @@
     "prepend-v0": "gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist/v0.js && gulp prepend-global --prod node_modules/AMP_CONFIG.json --local --target dist.3p/current-min/f.js"
   },
   "dependencies": {
+    "babel-polyfill": "6.26.0",
```

Comment:
Hopefully this isn't needed.

Reply:
unfortunately it is needed for `regenerator-runtime`

Reply:
I think regenerator only needs some symbol polyfills, which should be much smaller than the entire `babel-polyfill`.

Reply:
@jridgewell my expectation was `babel-preset-env` would have figured that out and added the polyfill but all the issues i found told me to include babel-polyfill. is there a specific package i can include that will allow me to remove babel-polyfill?

Reply:
We can either drop regenerator (loose generator support in Safari 9), or dig through the regenerator runtime code to see what we need.

Reply:
@jridgewell got it. do you mind if i create a separate issue/pr for that?

Reply:
Sure.